### PR TITLE
[tests] Improve coverage for schema.py

### PR DESCRIPTION
**Pull request by VSevagen**
Tests were missing for handling invalid datetime
values for enrollment_date and for some operators
in last_updated filter.

Signed-off-by: sevagenv <sevagenv@gmail.com>

### CODE REVIEW

File: tests/test_schema.py [hunk 1]
```diff
@@ -2399,6 +2399,35 @@ def test_filter_is_enrolled(self):
...
+        date_1 = '2002-01-01T00:00:00'
+        date_2 = '2001-01-01T00:00:00'
+
+        filter_invalid_format = "{}..{}".format(date_1, date_2)
+
+        client = graphene.test.Client(schema)
+        executed = client.execute(SH_INDIVIDUALS_ENROLLMENT_DATE_FILTER % filter_invalid_format,
+                                  context_value=self.context_value)
+
+        expected_error = 'Upper bound must be greater than the lower bound'
```
Comment (sduenas) [inline tests/test_schema.py:2426]: To keep the consistency on the style, please put long strings between double quotes.

File: tests/test_schema.py [hunk 2]
```diff
@@ -2399,6 +2399,35 @@ def test_filter_is_enrolled(self):
         indv = individuals[0]
         self.assertEqual(indv['mk'], 'c6d2504fde0e34b78a185c4b709e5442d045451c')
 
+    def test_filter_enrollment_date_invalid_date(self):
+      """ Check whether it fails when the filter has an invalid date """
```
Comment (sduenas) [inline tests/test_schema.py:2403]: Remove the leading and trailing spaces from the docstring.

**Review by sduenas — CHANGES_REQUESTED** [review]
Looks good but just some minor changes need to be addressed before merging it. These changes are needed to follow the code style we have in the project.

**Comment by VSevagen** [comment]
@sduenas I've updated the PR.

**Review by sduenas — APPROVED** [review]
LGTM

**Comment by VSevagen** [comment]
My bad, I forgot to run flake locally before pushing :sweat_smile: 
I updated it properly this time.

**Comment by sduenas** [comment]
> My bad, I forgot to run flake locally before pushing sweat_smile
> I updated it properly this time.

Can you rebase it before I merge it?

**Review by sduenas — APPROVED** [review]
LGTM

**Pull request merged**